lib/tasks/repo_export.rake

```diff
@@ -0,0 +1,9 @@
+desc "Run sufia export with Galter specific parameters"
```

Comment:
Just a convenience thing. I think it actually runs slow like this, but at least it stores the command we expect to run outside of our own heads.

Reply:
The command `sufia_export` loads the rails environment. Just curious if this could be an issue for the slowness? 

I think it would also help to be more specific here by adding "invenio" to the description.

Reply:
Ah, that might be it. I was thinking it might have to do with having rake add extra steps for making the call, particularly the `system` function from ruby. Honestly I don't think there is any way that this export can be fast so I'm not terribly concerned with it.